doc/api/index.md

```diff
@@ -169,6 +170,54 @@ player.loadVideo({
 });
 ```
 
+<a name="meth-reload"></a>
+### reload #####################################################################
+
+_arguments_:
+  - _options_ (``Object``)
+
+Re-load the last loaded content, with its original loadVideo options, as fast as
+possible, using the last loaded manifest. The aim is to reduce loading time by
+removing the fetching and parsing steps.
+
+At each new loadVideo call, the player fetches the manifest, and stores it as
```

Comment:
I don't get what this sentence brings. Is it to indicate implicitely that this API works after any loadVideo call? But the next sentence make it very clear.

Reply:
Maybe i can delete it, indeed